index.html

```diff
@@ -0,0 +1,187 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="src/shared/style/normalize.css" />
+    <link rel="stylesheet" href="src/shared/font/font.css" />
+    <link rel="stylesheet" href="src/shared/style/color.css" />
+    <link rel="stylesheet" href="src/shared/style/style.css" />
+    <title>판다마켓</title>
+  </head>
+  <body>
+    <!-- 헤더 -->
+    <header class="header">
+      <a href="/public" class="logoContainer">
+        <img
+          class="logoImg"
+          src="src/shared/img/logo/brand/logo.svg"
+          alt="logo"
+        />
+       <span class="logoButtonText">판다마켓</span>
+      </a>
+      <button class="loginButton" type="button" onclick="location.href='/login'">로그인</button>
```

Comment:
❗️ 수정요청
단순 페이지 이동일 시 a 태그를 사용해주세요! 스타일을 버튼처럼 작성하시면 됩니다~

```suggestion
      <a href="/login.html" class="loginButton">로그인</a>
```